.ci/README.md

```diff
@@ -0,0 +1,32 @@
+# Running a CI Deployment
```

Comment:
I'm OK having this doc here in the `.ci` dir, although I'd prefer having it in a place developers are able to find it in an easy manner, maybe creating a dedicated section `How to run the tests` in the main Readme. Nevertheless, this can be done in a follow up PR, given this one is intended to run on CI only